internal/volume/volumes_information.go

```diff
@@ -551,95 +550,124 @@ func (v *BackupVolumesInformation) generateVolumeInfoFromPVB() {
 	v.volumeInfos = append(v.volumeInfos, tmpVolumeInfos...)
 }
 
-// generateVolumeInfoFromDataUpload generate BackupVolumeInfo for DataUpload.
-func (v *BackupVolumesInformation) generateVolumeInfoFromDataUpload() {
-	if !features.IsEnabled(velerov1api.CSIFeatureFlag) {
-		v.logger.Debug("Skip generating BackupVolumeInfo when the CSI feature is disabled.")
-		return
+func (v *BackupVolumesInformation) getVolumeSnapshotClasses(
+	classes *snapshotv1api.VolumeSnapshotClassList,
+) (*snapshotv1api.VolumeSnapshotClassList, error) {
+	if classes != nil {
```

Comment:
Looks like we don't need this `classes` parameter and this check, as this function is not called for multiple times.

Reply:
looks like we have [similar feedback](https://github.com/vmware-tanzu/velero/pull/7974#discussion_r1674708795)?

Reply:
Thanks. Modified.